src/main/resources/_config.properties

```diff
@@ -19,11 +19,12 @@ DEMO.api_url=https://jsonplaceholder.typicode.com
 STAG.api_url=
 #=====================================================#
 
-#================ Locale Configurations ====================#
+#============== Locale Configurations ================#
 locale=en_US
 locale_auto_verification=false
 locale_encoding=UTF-8
 language=en_US
+browser_language=en_US
```

Comment:
remove it please. it affects a lot mobile web scenarios as of now and by default shouldn't be in config properties